python/mdvtools/dbutils/dbservice.py

```
@@ -10,10 +10,20 @@ class ProjectService:
     #  Routes -> /projects
     def get_active_projects():
         try:
-            f = [f[0] for f in ProjectService.failed_projects]
-            all = Project.query.filter_by(is_deleted=False).all()
-            projects = [p for p in all if p.id not in f]
-            return projects
+            failed_project_ids = {f[0] for f in ProjectService.failed_projects}  
+            projects = Project.query.filter(Project.is_deleted == False).all()
```

Ruff (0.8.2)

Avoid equality comparisons to `False`; use `if not Project.is_deleted:` for false checks

Replace with `not Project.is_deleted`

(E712)